.ci/azure-pipelines-api-client.yml

```diff
@@ -0,0 +1,72 @@
+﻿parameters:
+  - name: LinuxImage
+    type: string
+    default: "ubuntu-latest"
+  - name: GeneratorVersion
+    type: string
+    default: "4.3.1"
+
+jobs:
+- job: GenerateApiClients
+  displayName: 'Generate Api Clients'
+
+  pool:
+    vmImage: "${{ parameters.LinuxImage }}"
+
+  steps:
+    - task: DownloadPipelineArtifact@2
+      displayName: 'Download OpenAPI Spec Artifact'
+      inputs:
+        source: "specific"
+        artifact: "OpenAPI Spec"
+        path: "$(System.ArtifactsDirectory)/openapi"
+        project: "$(System.TeamProjectId)"
+        pipeline: "29" # The main server CI build
+        runVersion: "latestFromBranch"
+        runBranch: "refs/heads/$(System.PullRequest.TargetBranch)"
```

Comment:
If you want (and add the `dependsOn: Test` to this job, you can download from the current build. That might be more in line with what would be expected. (See the OpenAPI Spec SSH upload job for an example)

Reply:
Something like 
```yaml
  - task: DownloadPipelineArtifact@2
    displayName: 'Download OpenAPI Spec'
    inputs:
      source: 'current'
      artifact: "OpenAPI Spec"
      path: "$(System.ArtifactsDirectory)/openapispec"
      runVersion: "latest"
```

From the PR #4249